ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala

```diff
@@ -236,7 +239,7 @@ class CommandTrackerFlowTest
             Some(Status(Code.ABORTED.value)),
           )
         completionStreamMock.send(CompletionStreamElement.CompletionElement(completion))
-        results.requestNext().value shouldEqual completion
+        results.requestNext().value shouldEqual CompletionResponse(completion)
```

Comment:
Not sure I like that we're using the smart constructor here; I'd prefer we write the expected value out explicitly (and below).

Reply:
Agree, was just being lazy while fixing the test. will update